transactions/types.go

```diff
@@ -15,6 +15,13 @@ import (
 	wallet_common "github.com/status-im/status-go/services/wallet/common"
 )
 
+type SendTxArgsVersion uint
+
+const (
+	SendTxArgsVersion0 SendTxArgsVersion = 0
+	SendTxArgsVersion1 SendTxArgsVersion = 1
+)
```

Comment:
I guess this version will go away when deprecated code will be handle, maybe add a comment to remove this Version attribute?

Reply:
Sure, will add. And yes, once the mobile app switches to the new tx sending flow, and we make sure that there is no part of the app that uses old sendargs, we can simplify `Transactor` a lot and remove the version property.
Once we move all TXs sendings to this flow, we will have a uniform flow across the app that is easier to maintain and extend.